fastly/resource_fastly_service_dictionary_items_v1_test.go

```diff
@@ -292,38 +297,6 @@ func TestAccFastlyServiceDictionaryItemV1_batch_1001_items(t *testing.T) {
 	})
 }
 
-func TestAccFastlyServiceDictionaryItemV1_import(t *testing.T) {
```

Comment:
Can you clarify why this test is being removed? Is it replaced by the new test check above?

```
{
    ResourceName:      "fastly_service_dictionary_items_v1.items",
    ImportState:       true,
    ImportStateVerify: true,
},
```

Reply:
The import tests have now been folded into the create named tests as a step and a means to confirm that configuration can be imported.  This was raised through the linter.